pylxd/tests/test_container.py

```diff
@@ -141,6 +141,13 @@ def test_delete(self):
 
         an_container.delete(wait=True)
 
+    def test_migrate(self):
+        """A container is migrate."""
+        name = 'an-container'
+
+        an_container = container.Container.get(self.client, name)
+        an_container.migrate()
```

Comment:
This isn't a good test, as it doesn't return assert anything (I realize that we have _some_ tests that don't assert things, but we have a mock service that has some support, or the test would fail).